jre/java/java/util/stream/DoubleStream.java

```diff
@@ -202,6 +202,39 @@ static DoubleStream of(double t) {
 
   DoubleStream distinct();
 
+  default DoubleStream dropWhile(DoublePredicate predicate) {
```

Comment:
I commenting in this file but let's finish up review/refactoring here until we finalize so you won't need keep updating all files until we are done.